File: Selenium/Activities/Activity19.java
package activities;
import org.openqa.selenium.Alert;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import io.github.bonigarcia.wdm.WebDriverManager;

public class Activity19 {
    public static void main(String[] args){
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://v1.training-support.net/selenium/javascript-alerts");
        System.out.println("Title is :" +driver.getTitle());
        driver.findElement(By.id("confirm")).click();
        Alert Confirm_alert = driver.switchTo().alert();
        System.out.println("Text in Confirm Alert is :" +Confirm_alert.getText());
        Confirm_alert.accept();
        driver.findElement(By.id("confirm")).click();
        Confirm_alert.dismiss();
        driver.close();
    }
}

File: Selenium/Activities/Activity19.py
import time

from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from selenium.webdriver.common.alert import Alert
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())
with webdriver.Firefox(service=service) as driver:
    driver.get("https://v1.training-support.net/selenium/javascript-alerts")
    print("Title is :" ,driver.title)
    alert_element = driver.find_element(By.ID,"confirm")
    alert_element.click()
    confirm_alert = driver.switch_to.alert
    print("text in alert is :",confirm_alert.text)
    time.sleep(2)
    confirm_alert.accept()
    alert_element.click()
    time.sleep(2)
    confirm_alert.dismiss()
    driver.quit()